app/templates/suppliers/_frameworks_live.html

```diff
@@ -29,8 +29,14 @@
         </p>
       {% endif %}
     {% endcall %}
+
     {% if framework.onFramework and not framework.needs_to_complete_declaration %}
-      {{ summary.edit_link('View documents and ask a question', url_for('.framework_dashboard', framework_slug=framework.slug)) }}
+        {% call summary.field() %}
+            {% if framework.framework == 'digital-outcomes-and-specialists' %}
+                <a href="{{ url_for('.opportunities_overview', framework_slug=framework.slug) }}">View your opportunities</a><br>
```

Comment:
"View your opportunities" feels a bit vague - it would be nice to be more precise that these are the opportunities that you have previously applied for, because I don't think that's obvious from the link text.

And if suppliers click this and see an empty list - as many will if they haven't applied for work recently - it won't be obvious why there's nothing there for them.